Implemented "Current selection" group option (#11455)
JabRef/jabref: CHANGELOG.md

- We added an exporter and improved the importer for Endnote XML format. [#11137](https://github.com/JabRef/jabref/issues/11137)
- We added support for using BibTeX Style files (BST) in the Preview. [#11102](https://github.com/JabRef/jabref/issues/11102)
- We added support for automatically update LaTeX citations when a LaTeX file is created, removed, or modified. [#10585](https://github.com/JabRef/jabref/issues/10585)

> - We added the ability to collect a group by your current selection. [#11449](https://github.com/JabRef/jabref/issues/11449)

Reviewer: ```suggestion
- We added the ability to add the current selection to a newly created group. [#11449](https://github.com/JabRef/jabref/issues/11449)
```
